File: src/main/java/org/polyfrost/overflowanimations/command/OldAnimationsCommand.java
package org.polyfrost.overflowanimations.command;

import org.polyfrost.oneconfig.api.commands.v1.factories.annotated.Command;
import org.polyfrost.oneconfig.utils.v1.dsl.ScreensKt;
import org.polyfrost.overflowanimations.config.OldAnimationsSettings;

@Command(value = {"overflowanimations", "oam", "oldanimations", "animations"})
public class OldAnimationsCommand {
    @Command
    public void main() {
        ScreensKt.openUI(OldAnimationsSettings.INSTANCE);
    }
}

File: src/main/kotlin/org/polyfrost/overflowanimations/command/OldAnimationsCommand.kt
package org.polyfrost.overflowanimations.command

import cc.polyfrost.oneconfig.utils.commands.annotations.Command
import cc.polyfrost.oneconfig.utils.commands.annotations.Main
import org.polyfrost.overflowanimations.OverflowAnimations
import org.polyfrost.overflowanimations.config.MainModSettings

@Command(
    value = OverflowAnimations.MODID,
    aliases = ["oam", "oldanimations", "animations"],
    description = "Access the " + OverflowAnimations.NAME + " GUI."
)
class OldAnimationsCommand {

    @Main
    fun handle() {
        MainModSettings.openGui()
    }

}
